release-notes/7.0/preview/api-diff/preview2/Microsoft.WindowsDesktop.App/7.0-preview2_System.Diagnostics.md

```diff
@@ -0,0 +1,9 @@
+# System.Diagnostics
```

Comment:
It's surprising to see this diff as part of the Windows Desktop

Reply:
@ericstj @joperezr do you know why the `Microsoft.WindowsDesktop.App.Ref` nupkg contains System.Diagnostics and System.Drawing APIs? Is that expected?

Reply:
Yes, those assemblies are distributed as part of the WindowsDesktop shared framework so they are expected to show up here.  Winforms/WPF include these Windows specific packages in their shared framework due to dependencies in their API and implementation assemblies.

Reply:
Ok great, thanks for the clarification.

Reply:
Makes sense. Thank you.